### ISSUE TITLE
[Quest] The Wayward Apprentice

### ISSUE THREAD

**Issue by Luminette** [FeenixServerProject/Archangel_2.4.3_Bugtracker#1243]
Wowhead: http://www.wowhead.com/quest=9254

After completing all the quest's in Fairbreeze Village I got this quest.Went there NPC is present but you cant interact with it,IE,no cant deliver.I tryed abbandoning/retake,clear wtd and cashe repeat first step,do all the quest's and come back.
Screenshot: http://tinypic.com/r/ekq1cp/6

**Comment by mrsmahlstedt** [comment]
Still occurring as of 17/10/2015. Unable to complete quest as Blood Elf or Undead. I was able to complete this quest a few months back, but it seems to be broken again.

**Comment by devastator** [comment]
Should be fixed in npc_apprentice_mirveda script

**Comment by Myzandria** [comment]
Nope, not working... ... as of 4 years later... cricket cricket...

**Comment by devastator** [comment]
Issue found and fixed. Soon on PTR
